.github/workflows/olm_tests.yaml

```diff
@@ -53,12 +53,12 @@ jobs:
       run: |
         # TODO: Use operator-sdk run bundle and bundle-upgrade commands for
         # testing with OLM.
-        PREV_BUNDLE_INDEX_IMG_VERSION=$(make prev-bundle-index-image-version)
+        REPLACES_VERSION=$(make get-replaces-version)
         OPERATOR_VERSION=$(awk '/^VERSION \?=/ {print $3}' Makefile)
         REPO=localhost:5000
         IMG=${REPO}/gatekeeper-operator:${GITHUB_SHA}
-        PREV_BUNDLE_INDEX_IMG=quay.io/gatekeeper/gatekeeper-operator-bundle-index:${PREV_BUNDLE_INDEX_IMG_VERSION}
-        make build-and-push-bundle-images IMG=${IMG} REPO=${REPO} VERSION=${GITHUB_SHA} OPERATOR_VERSION=${OPERATOR_VERSION} PREV_BUNDLE_INDEX_IMG=${PREV_BUNDLE_INDEX_IMG}
+        PREV_BUNDLE_INDEX_IMG=quay.io/gatekeeper/gatekeeper-operator-bundle-index:${REPLACES_VERSION}
+        make build-and-push-bundle-images IMG=${IMG} REPO=${REPO} VERSION=${GITHUB_SHA} OPERATOR_VERSION=${OPERATOR_VERSION}
```

Comment:
@JustinKuli the `get-replaces-version` is used here to grab it for a var to specify the `PREV_BUNDLE_INDEX_IMG` which itself is referenced in the Makefile. Yet more indirection that I didn't want to untangle right now.